Importer.php

```diff
@@ -123,9 +124,19 @@ class Importer
      */
     private $isMainImport = true;
 
+    /**
+     * @var ApiQuotaHelper
+     */
+    private $apiQuotaHelper;
+
+    /**
+     * @var int
+     */
+    private $maxAvailableQueries = 0;
```

Comment:
@elabuwa Similar changes are required in `ImporterGA4.php` too.

Reply:
Done @AltamashShaikh